sound/soc/sof/trace.c

```diff
@@ -8,11 +8,14 @@
 // Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
 //
 
+#include <linux/jiffies.h>
 #include <linux/debugfs.h>
 #include <linux/sched/signal.h>
 #include "sof-priv.h"
 #include "ops.h"
 
+u64 shift;
```

Comment:
we should avoid such local variables, let's move this in sof-priv.h

Reply:
@plbossart got it